commands/deploy.go

```diff
@@ -21,9 +23,18 @@ const MaxFileSize = 1073741824 // 1GB
 
 // DeployCmd handles deploying an app to Section.
 type DeployCmd struct {
-	Debug     bool
-	Directory string   `default:"."`
-	ServerURL *url.URL `default:"https://aperture.section.io/new/code_upload/v1"`
+	AccountID        int `default:"4322"`  // harc-coded until authentication is implemented
+	AppID            int `default:"65443"` // hard-coded for now until authentication is implmented
+	Debug            bool
+	Directory        string   `default:"."`
+	ServerURL        *url.URL `default:"https://aperture.section.io/new/code_upload/v1"`
+	ApertureURL      string   `default:"https://aperture.section.io/api/v1"`
+	EnvUpdatePathFmt string   `default:"/account/%d/application/%d/environment/%s/update"`
```

Comment:
And I will refactor this out into the `api` package, so the deploy command doesn't need to know the internals of how this request works.